docs/submitting-applications.md

```diff
@@ -99,7 +105,7 @@ run it with `--help`. Here are a few examples of common options:
   /path/to/examples.jar \
   1000
 
-# Run on a YARN cluster
+# Run on a YARN cluster without --deploy mode
```

Comment:
Same -- name was fine here. This needs to use `--deploy-mode` if that's the primary, recommended invocation right?